File: Python/Warmup-1/monkey_trouble.py

# this function return true (trouble) if two monkeys are smiling or two monkeys are not smiling
def monkey_trouble(a_smile, b_smile):
	b = False

	if (a_smile and b_smile) or (not a_smile and not b_smile):
		b = True;

	return b


print(monkey_trouble(True, True))
print(monkey_trouble(False, False))
print(monkey_trouble(True, False))

File: JavaScript/Warmup-1/monkeyTrouble.js

// we have two monkeys, a and b, and the parameters aSmile and bSmile indicate if each is smiling
// we are in trouble if they are both smiling or if neither of them is smiling
// return true if we are in trouble
const monkeyTrouble = (aSmile, bSmile) => {

    return ((aSmile && bSmile) || (!aSmile && !bSmile)) ? true : false;

}

console.log(monkeyTrouble(true, true));
console.log(monkeyTrouble(false, false));
console.log(monkeyTrouble(true, false));
